higlass/server.py

```diff
@@ -323,7 +323,8 @@ class Server:
     diskcache_directory = "/tmp/hgflask/dc"
 
     def __init__(
-        self, tilesets, port=None, host="localhost", tmp_dir="/tmp/hgflask"
+        self, tilesets, port=None, host="localhost", tmp_dir="/tmp/hgflask",
```

Comment:
Just out curiosity, why did you decide to add a type to `no_fuse` but not `host`?

Reply:
I didn't add `host`, that arg was already there.